packages/manager/modules/dedicated-universe-components/src/validator/validator.service.js

```diff
@@ -0,0 +1,108 @@
+export default class Validator {
```

Comment:
```js
import punycode from 'punycode';
```

Reply:
In other places `punycode` is set as a global (https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/.eslintrc.json#L9)